content/05.main-text.md

```diff
@@ -749,10 +767,68 @@ Producing results for these queries relies on internal calls to the PostgreSQL r
 They were designed to power the hetnet connectivity search webapp,
 but are also available for general research use.
 
-### Webapp & Frontend
+### Hetio Website
 
-TODO: write this section.
-https://github.com/greenelab/connectivity-search-manuscript/issues/29
+We created a static website to serve as the home for the Hetio organization using Jekyll.
+The source code is available in the [`het.io`](https://github.com/hetio/het.io) repository.
+
+To ease the burden of maintenance and typical website hosting costs, the HTML, CSS, JavaScript, and other assets for the website are hosted for free on GitHub Pages.
+Jekyll was chosen over other static site generators for simplicity, ease-of-use, popularity (support), and its convenient integration with GitHub Pages.
+To make a change to the website, an author simply commits the changes (either directly or through a pull request) to the repository's `gh-pages` branch, and GitHub automatically re-compiles the website and hosts the resulting files at the provided custom domain url.
+No explicit build instructions or other continuous integration is required.
+
+### Visual Design
```

Comment:
Should these be heading 4 to indicate that they are under Hetio Website?

Reply:
The visual design is kind of even more global than the hetio website imo, since the colors are used in representing the knowledge graph itself and various materials outside of the website. The webapp section could be under the website though. Or maybe an organization like this:

```markdown
### Backend and API

### Visual Design

### Frontend

#### Hetio Website

#### Webapps
```

I guess visual design could go under frontend in this organization since it's just "user-facing".

Reply:
Open to anything here. I'd probably put Visual Design at the end so Backend and Frontend are next to each other. No strong preference for whether Visual Design should be heading 3 or 4.